Packs/CommonScripts/Scripts/NumberOfPhishingAttemptPerUser/CHAGELOG.md

```diff
@@ -1,3 +0,0 @@
-## [Unreleased]
-- Added NumberOfPhishingAttemptPerUser automation. Tagged as dynamic-section. Shows a bar chart of the number of incident the 'To' and 'From' email addresses
-  appeared on.
```

Comment:
why did you remove this

Reply:
Because the name was missing an N and there is already a new changelog